scss/detail.scss

```diff
@@ -1,6 +1,11 @@
 .book-container {
+  background-color: $white;
+  -webkit-box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
```

Comment:
Just a quick comment. You can name this as variable by doing `$black-05` that tells us that it will have opacity

Reply:
You have the reason, thanks!